packages/portal/src/components/stories/StoriesInterface.vue

```diff
@@ -243,6 +246,12 @@
         }
       },
 
+      setFocusToTypeFilter() {
```

Comment:
Let's instead call `blur `on `esc` in StoriesTagsDropdown.

Reply:
I've pushed a commit to make that so. I think that's more correct and cleaner.

Reply:
This doesn't allow a keyboard only user to get past the tag search box without tabbing through all the options.
I think moving the focus to the following element is better UX overall.

Reply:
The issue with moving focus only exists in firefox. It is possible to get around by hitting esc twice, which moves the focus outside of the input and let's the user tab to the story type toggle.

Reply:
After some testing and discussions this mainly seems a bit of an issue in FF. Other latest browsers seem to keyboard navigate as expected.